paymaster-analysis/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "@safe-global/paymaster-analysis",
```

Comment:
since this is never intended to be published, I'd add a `private` flag to prevent accidental publishing

Reply:
Also, I'm not sure if paymaster analysis is an accurate title because it validates both paymaster and non-paymaster scenarios. Perhaps something like `4337-gas-metering` is more accurate

Reply:
Done 👍🏾